Onboarding/menu-personalised.md

```diff
@@ -0,0 +1,673 @@
+
+card Practise, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
+
+card Generic1, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
+
+card Generic2, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
+
+card Generic3, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
+
+card MainMenu, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
+
+```
+
+```stack
+card DomainShowcase do
+  log("DomainShowcase")
+end
+
+card MainMenu do
+  log("MainMenu")
+end
+
+```
+
+## View topics for you
+
+```stack
+card LibraryTopics, then: DisplayLibraryTopics do
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "mnch_onboarding_library"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  page_id = search.body.results[0].id
+
+  content_data =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  message = content_data.body.body.text.value
+  menu_items = map(message.list_items, & &1.value)
+end
+
+card DisplayLibraryTopics, then: DisplayLibraryTopicsError do
+  selected_topic =
+    list("Select option",
+      LoveRelationship: "@menu_items[0]",
+      PregnancyInfo: "@menu_items[1]",
+      BabyChildHealth: "@menu_items[2]",
+      WellBeing: "@menu_items[3]",
+      FamilyPlanning: "@menu_items[4]",
+      HealthProfessional: "@menu_items[5]",
+      MainMenu: "@menu_items[6]"
+    ) do
+      text("@message.message")
+    end
+end
+
+card DisplayLibraryTopicsError, then: DisplayLibraryTopicsError do
+  selected_topic =
+    list("Select option",
+      LoveRelationship: "@menu_items[0]",
+      PregnancyInfo: "@menu_items[1]",
+      BabyChildHealth: "@menu_items[2]",
+      WellBeing: "@menu_items[3]",
+      FamilyPlanning: "@menu_items[4]",
+      HealthProfessional: "@menu_items[5]",
+      MainMenu: "@menu_items[6]"
+    ) do
+      text("@list_error_text")
+    end
+end
+
+card LoveRelationship, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
+
+card PregnancyInfo, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
+
+card BabyChildHealth, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
+
+card WellBeing, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
+
+card FamilyPlanning, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
+
+card HealthProfessional, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
+
+card MainMenu, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
```

Comment:
Don't think you need this? I think the navigation here should direct the user back to the `PersonalisedMenu`?